charts/oes/values.yaml

```diff
@@ -628,37 +677,37 @@ spinnaker:
         healthEndpoint: /health
         kubernetes:
           useExecHealthCheck: false
-        artifactId: quay.io/opsmxpublic/ubi8-spin-gate:1.16.1
+        artifactId: quay.io/opsmxpublic/ubi8-spin-gate:1.18.1
       deck.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-deck:3.3.0
+        artifactId: quay.io/opsmxpublic/ubi8-spin-deck:1.22.2-mj
       clouddriver.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.9.2
+        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.11.1
       clouddriver-caching.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.9.2
+        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.11.1
       clouddriver-rw.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.9.2
+        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.11.1
       clouddriver-ro.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.9.2
+        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.11.1
       clouddriver-ro-deck.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.9.2
+        artifactId: quay.io/opsmxpublic/ubi8-spin-clouddriver:6.11.1
       echo.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-echo:version-2.12.1
+        artifactId: quay.io/opsmxpublic/ubi8-spin-echo:2.14.0
       echo-scheduler.yml:
-        artifactId: quay.io/opsmxpublic/ubi8-spin-echo:version-2.12.1
```

Comment:
If we are defining ubi8-spin-* images in both values.yaml and values-oss.yaml. What's the different between two?
I suppose we should be using ubi8-oes-* images in values.yaml. For example, ubi8-oes-echo:<tag>

Reply:
Took care of this